doc/3/controllers/document/update/index.md

```diff
@@ -0,0 +1,63 @@
+---
+code: true
+type: page
+title: update
+description: Updates a document
+---
+
+# update
+
+Updates a document.
+
+---
+
+## Arguments
+ 
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> update(
+      final String index,
+      final String collection,
+      final String id,
+      final ConcurrentHashMap<String, Object> document)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, Object>> update(
+      final String index,
+      final String collection,
+      final String id,
+      final ConcurrentHashMap<String, Object> document,
+      final ConcurrentHashMap<String, Object> options)
```

Comment:
We should have a `DocumentOptions` class containing `waitForRefresh`, `retryOnConflict` and `source`